src/resources/parser/glb-parser.js

```diff
@@ -971,7 +971,7 @@ var createMaterial = function (gltfMaterial, textures, disableFlipV) {
             color = specData.diffuseFactor;
             // Convert from linear space to sRGB space
             material.diffuse.set(Math.pow(color[0], 1 / 2.2), Math.pow(color[1], 1 / 2.2), Math.pow(color[2], 1 / 2.2));
-            material.opacity = (color[3] != null) ? color[3] : 1;
+            material.opacity = color[3];
```

Comment:
was this change intentional?

Reply:
Yes. The spec states that `diffuseFactor` is a 4-element array. I don't see a circumstance where the fourth element can be `null` or `undefined`. Am I missing something?

Reply:
See here: https://kcoley.github.io/glTF/extensions/2.0/Khronos/KHR_materials_pbrSpecularGlossiness/#specular---glossiness

Reply:
Makes sense.